google-cloud-speech/lib/google/cloud/speech/audio.rb

```diff
@@ -260,15 +265,15 @@ def recognize max_alternatives: nil, profanity_filter: nil, phrases: nil
         #   results = op.results
         #
         def process max_alternatives: nil, profanity_filter: nil,
-                    phrases: nil
+                    phrases: nil, word_times: nil
```

Comment:
K, I'll change this unless someone else feels strongly that `word_times` is better.